resources/shaders/generate_mips_cs.hlsl

```diff
@@ -0,0 +1,22 @@
+Texture2D<float4> SrcTexture : register(t0);
```

Comment:
Follow the c++ style guide for hlsl as well pleaze.

Reply:
Done 👍